README.adoc

```diff
@@ -22,5 +22,39 @@ image:img/screenshot.png[alt="screenshot"]
 * Multi-PCB feature (different PCB variants of the same schematic)
 * Automatic netlist synchronisation between schematic and board
 
-*You want to give LibrePCB a try?* +
+
+[#projectstatus]
+== Project Status
+
+LibrePCB is still under development and many things are not yet
+working as expected. So here an overview about the project status:
+
+=== What's working (more or less)
+
+* Downloading, creating and modifying libraries
+* Drawing simple schematics with multiple sheets
+* Creating *very* simple PCBs (incl. Gerber export)
+
+=== What's not working (or even missing)
+
+* Schematic editor:
+** No hierarchical sheets
+** No buses
+** No BOM export
+* Board editor:
+** No airwires
+** No texts (e.g. on silkscreen)
```

Comment:
texts -> text

Reply:
Sure? "No airwires" and "No planes" is also plural, so why use singular for "text"?

Reply:
I think it's a bit unusual to use "texts" in plural, similar to "information". What do you think @rnestler?

Reply:
I would say in this context "text" means an object on the board, and there can be multiple of them, so the plural form is fine here. See also:
- https://en.wiktionary.org/wiki/texts#English
- https://www.wordhippo.com/what-is/the-plural-of/text.html

Reply:
As a compromise I'd suggest "No text objects"

Reply:
too late ;)